LAD-AMA-Common/telegraf_utils/telegraf_config_handler.py

```diff
@@ -229,35 +251,41 @@ def parse_config(data, me_url, mdsd_url, is_lad, az_resource_id, subscription_id
                         metricsext_rename_str += "\n" + " "*2 + "[[processors.rename.replace]]\n"
                         metricsext_rename_str += " "*4 + "field = \"" + field + "\"\n"
                         metricsext_rename_str += " "*4 + "dest = \"" + plugin + "/" + field + "\"\n"
+                elif is_vmi:
+                    # no rename of fields as they are set in telegraf directly
+                elif is_vmi_rate_counter:
+                    metricsext_rename_str += "\n" + " "*2 + "[[processors.rename.replace]]\n"
```

Comment:
Wouldnt we need to rename the namespace for these as well?

Reply:
for non-rate VMI counters:
- rename namespace for ME

for rate VMI counters:
- rename namespace and field for ME
- rename namespace and field for mdsd